### ISSUE TITLE
Cursor not visible on resolution higher than 1080p

### ISSUE THREAD

**Issue by SHIRO-Suit** [anegostudios/VintageStory-Issues#3740]
### Game Version

v1.19.4

### Platform

Windows

### Modded

Vanilla

### SP/MP

Singleplayer

### Description

Launching the game in fullscreen with a resolution higher than 1080p makes the mouse disapear.
I tried with a 1440p monitor and a 4k one. same result each time.
To be more accurate, just switching to fullscreen keeps the cursor visible, but as soon as I move it one pixel it vanishes. (I don't know if it's of any help)

### How to reproduce

1. Set your monitor to a resolution of 2K or more.
2. Run the game and press F11 to activate fullscreen.

### Logs

```Log```

**Comment by radfast** [comment]
Please post up your full `clientsettings.json`  (recommended to remove your personal info from the `stringSettings` section, so remove `mptoken`, `sessionkey`, `sessionsignature`, `useremail`)

The devs all have 1440p or 4k monitors with no issues, so wondering what might be different on your setup

**Comment by SHIRO-Suit** [comment]
Thank you for your time,
I did a bunch of other tests with one working but not really being a fix.
I have a dual monitor setup with 2k@165Hz and 4k@60Hz. 
Disabling the 4K one while I play with the 2k fixes the issue. (but I need it to stream)
but, setting the other one to principal, changing orientation or arrangement, changing framerate of the 2k to 60Hz, didn't work.

as for the json, here it is : 
```
{
  "keyMapping": {},
  "dialogPositions": {},
  "stringSettings": {
    "settingsVersion": "1.12",
    "language": "en",
    "defaultFontName": "sans-serif",
    "decorativeFontName": "Lora",
    "recordingCodec": "rawv",
    "glContextVersion": "3.3",
// Sensitive Data
    "entitlements": null,
    "masterserverUrl": "https://masterserver.vintagestory.at/api/v1/servers/",
    "modDbUrl": "https://mods.vintagestory.at/",
    "audioDevice": null,
    "playeruid": "dk01DkYHRf5FZ6yiG+kh3eto",
    "playername": "Searaw",
    "lastSkinSelection": "baseskin:skin5,eyecolor:purple,underwear:breeches,voicetype:harmonica,voicepitch:low,hairbase:messy2,hairextra:thicklongfrenchbraid,facialexpression:smirk,mustache:none,beard:none,haircolor:ecru",
    "currentHandbookCategoryCode": null
  },
  "intSettings": {
    "fpHandsFoV": 75,
    "mipmapLevel": 3,
    "musicFrequency": 2,
    "graphicsPresetId": 11,
    "webRequestTimeout": 10,
    "maxAnimatedElements": 46,
    "shadowMapQuality": 2,
    "optimizeRamMode": 1,
    "ssaoQuality": 1,
    "viewDistance": 256,
    "fieldOfView": 80,
    "maxFps": 60,
    "particleLevel": 100,
    "maxDynamicLights": 25,
    "weirdMacOSMouseYOffset": 5,
    "screenshotExifDataMode": 0,
    "chunkVerticesUploadRateLimiter": 0,
    "soundLevel": 100,
    "entitySoundLevel": 100,
    "ambientSoundLevel": 100,
    "weatherSoundLevel": 100,
    "musicLevel": 4,
    "screenWidth": 2480,
    "screenHeight": 597,
    "gameWindowMode": 1,
    "rightDialogMargin": 0,
    "leftDialogMargin": 0,
    "maxQuadParticles": 8000,
    "maxCubeParticles": 4000,
    "maxAsyncQuadParticles": 80000,
    "maxAsyncCubeParticles": 80000,
    "itemCollectMode": 0,
    "mouseSmoothing": 30,
    "mouseSensivity": 50,
    "modelDataPoolMaxVertexSize": 500000,
    "modelDataPoolMaxIndexSize": 750000,
    "modelDataPoolMaxParts": 1500,
    "maxTextureAtlasWidth": 4096,
    "maxTextureAtlasHeight": 2048,
    "ambientBloomLevel": 20,
    "godRays": 0,
    "chatWindowWidth": 700,
    "chatWindowHeight": 200,
    "recordingBufferSize": 60,
    "vsyncMode": 2,
    "windowBorder": 0,
    "guiColorsPreset": 1,
    "schematicMaxUploadSizeKb": 200
  },
  "boolSettings": {
    "selectedBlockOutline": true,
    "showMoreGfxOptions": true,
    "multipleInstances": false,
    "showSurvivalHelpDialog": true,
    "showCreativeHelpDialog": true,
    "smoothShadows": true,
    "flipScreenshot": true,
    "renderClouds": true,
    "renderParticles": true,
    "transparentRenderPass": true,
    "extendedDebugInfo": false,
    "showentitydebuginfo": false,
    "showBlockInfoHud": true,
    "showBlockInteractionHelp": true,
    "showCoordinateHud": false,
    "wavingStuff": true,
    "renderMetaBlocks": false,
    "ambientParticles": true,
    "pauseGameOnLostFocus": false,
    "viewBobbing": true,
    "invertMouseYAxis": false,
    "highQualityAnimations": true,
    "occlusionculling": true,
    "developerMode": false,
    "glDebugMode": false,
    "showWhitelistedServers": true,
    "showModdedServers": true,
    "showPasswordProtectedServers": true,
    "showOpenForAllServers": true,
    "liquidFoamAndShinyEffect": true,
    "testGlExtensions": true,
    "offThreadMipMaps": false,
    "toggleSprint": false,
    "separateCtrlKeyForMouse": false,
    "allowSettingHRTFaudio": true,
    "useHRTFaudio": false,
    "force48khzHRTFaudio": true,
    "fxaa": true,
    "autoChat": true,
    "bloom": true,
    "skipNvidiaProfileCheck": false,
    "scaleScreenshot": false,
    "immersiveMouseMode": false,
    "startupErrorDialog": false,
    "hasGameServer": false,
    "directMouseMode": false,
    "showMinimapHud": true,
    "chatdialogvisible": true
  },
  "floatSettings": {
    "guiScale": 1.125,
    "fontSize": 1.0,
    "lodBias": 0.33,
    "lodBiasFar": 0.67,
    "blockAtlasSubPixelPadding": 0.01,
    "itemAtlasSubPixelPadding": 0.0,
    "gammaLevel": 2.2,
    "extraGammaLevel": 1.0,
    "brightnessLevel": 1.0,
    "sepiaLevel": 0.2,
    "cameraShakeStrength": 1.0,
    "wireframethickness": 1.0,
    "previewTransparency": 0.3,
    "fpHandsYOffset": 0.0,
    "mouseWheelSensivity": 1.0,
    "recordingFrameRate": 30.0,
    "gameTickFrameRate": -1.0,
    "instabilityWavingStrength": 1.0,
    "ssaa": 1.0,
    "megaScreenshotSizeMul": 2.0,
    "minbrightness": 0.0
  },
  "stringListSettings": {
    "multiplayerservers": [],
    "disabledMods": [],
    "dialogPositions": [],
    "modPaths": [
      "Mods",
      "C:\\Users\\shiro\\AppData\\Roaming\\VintagestoryData\\Mods"
    ],
    "customPlayStyles": []
  }
}
```

**Comment by SHIRO-Suit** [comment]
And the cursor is always visible in OBS even with the texture its supposed to have in the game

**Comment by SHIRO-Suit** [comment]
Nevermind, it was on my end, sorry for assuming it was a bug with the game.

I just noticed all my games did this in fullscreen, its been a while since I played on my computer so I hadn't noticed.
It was a combination of Compatibility mode set to Windows 8 and something else that fixed the issue for me.

Just windows 8 did not work, and I tried a lot of things so I don't know exactly what fixed it in between.

Thank you for your help anyway and sorry again.
